.github/workflows/release.yml

```diff
@@ -118,3 +118,11 @@ jobs:
             $(printf -- "--tag ${{ vars.DOCKER_REPO }}:%s " ${tags[@]}) \
             $(printf "${{ vars.DOCKER_REPO }}@%s " ${digests[@]})
           echo "::endgroup::"
+
+  chart-release:
```

Comment:
you'll need to add `environment: release` for this to work now that the secrets have been created in this environment.

Reply:
excellent point, done!

The new key fingerprint is `dU/0iLRxlQu5Apyx8y16eVPqhsClH2BMCQiMdEEcyBU=`.